rest-dashboard_integration_test.go

```diff
@@ -31,6 +31,10 @@ func Test_Dashboard_CRUD(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if _, err = client.DeleteDashboardByUID(ctx, board.UID); err != nil {
```

Comment:
This is nice that it tests whether it succeeds deleting a board that has been deleted already. However, it doesn't actually check whether the deletion (the action) works. Could we amend this by creating two dashboards and deleting them in two different ways?

Reply:
I believe I addressed your comment.  I was trying to understand the logic.  Just confirming, you start the test by deleting in case the dashboard already exists? 

Let me know how the new test looks like and if it validates your ask.